recipes/agate-sql/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{%set name = "agate-sql" %}
+{%set version = "0.5.0" %}
+{%set compress_type = "tar.gz" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "e5fa728dd5c0d71d3c7653a794cd830330fa2cf8b2f02230db68ddea3ab32ffb" %}
+{%set build_num = "0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ compress_type }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ compress_type }}
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: {{ build_num }}
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - agate >=1.5.0
```

Comment:
Can you try removing the `run` time dependencies from the `build` section and see how it goes?